enterprise-suite/values.yaml

```diff
@@ -17,6 +17,38 @@ configMapReloadVersion: v0.2.2
 kubeStateMetricsVersion: v1.2.0
 # prom/node-exporter
 nodeExporterVersion: v0.15.2
+# janeczku/go-dnsmasq
+goDnsmasqVersion: release-1.0.7
+# alpine
+alpineVersion: latest
+# busybox
+busyboxVersion: latest
+
+#################################################
+# Image names to use. Combined with the versions above, gives the fully qualified docker image name.
+#
+# es-console image
+esConsoleImage: '{{ .Values.imageCredentials.registry }}/enterprise-suite/es-console'
+# es-monitor-api image
+esMonitorImage: '{{ .Values.imageCredentials.registry }}/enterprise-suite/es-monitor-api'
+# es-grafana image
+esGrafanaImage: '{{ .Values.imageCredentials.registry }}/enterprise-suite/es-grafana'
+# prometheus image
+prometheusImage: prom/prometheus
+# alertmanager image
+alertManagerImage: prom/alertmanager
+# configmap-reload image
+configMapReloadImage: jimmidyson/configmap-reload
+# kube-state-metrics image
+kubeStateMetricsImage: gcr.io/google_containers/kube-state-metrics
+# node-exporter image
+nodeExporterImage: prom/node-exporter
+# go-dnsmasq image
+goDnsmasqImage: janeczku/go-dnsmasq
+# alpine image
+alpineImage: alpine
```

Comment:
I guess we use latest for this and busybox now, but do we want to consider specifying particular versions?

Reply:
Yes we should!

Reply:
I'll make that change.